tests/mshevchuk/homework_1.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+
+def test_max():
+    assert max((2, 3, 1, -2, -5, -8, -12), key=abs) == -12
+
+
+def test_max1():
+    assert max((2, 3, 1, -2, -5, -8, -12), key=abs) is not -12
+
+
+def test_abs():
+    assert abs(-12) != -12
+
+
+def test_compare_tuples1():
+    assert ((1, 2 + 2, 3) == (1, 4, 3)) is True
+
+
+def test_compare_tuples2():
+    assert ((1, 2) > (1, 1, 1, 1)) is True
```

Comment:
here you can avoid is True part
since assert will match expression in parenthesis with bool True